R/build-articles.R

```diff
@@ -264,8 +264,8 @@ build_article <- function(name,
 
       options <- list(
         template = template$path,
-        self_contained = FALSE,
-        theme = NULL
+        self_contained = FALSE
```

Comment:
For safety (since this will potentially break other formats), how about only not setting theme when the author is using `bookdown::html_vignette()`?

Reply:
Ok, I've read the source code of RMarkdown to see what `theme` does, and it looks like it should be safe to always omit it.